src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -173,12 +172,26 @@ public static class StandardLearnersCatalog
             float learningRate = AveragedLinearArguments.AveragedDefaultArgs.LearningRate,
             bool decreaseLearningRate = AveragedLinearArguments.AveragedDefaultArgs.DecreaseLearningRate,
             float l2RegularizerWeight = AveragedLinearArguments.AveragedDefaultArgs.L2RegularizerWeight,
-            int numIterations = AveragedLinearArguments.AveragedDefaultArgs.NumIterations,
-            Action<AveragedPerceptronTrainer.Arguments> advancedSettings = null)
+            int numIterations = AveragedLinearArguments.AveragedDefaultArgs.NumIterations)
         {
             Contracts.CheckValue(ctx, nameof(ctx));
             var env = CatalogUtils.GetEnvironment(ctx);
-            return new AveragedPerceptronTrainer(env, labelColumn, featureColumn, weights, lossFunction ?? new LogLoss(), learningRate, decreaseLearningRate, l2RegularizerWeight, numIterations, advancedSettings);
+            return new AveragedPerceptronTrainer(env, labelColumn, featureColumn, weights, lossFunction ?? new LogLoss(), learningRate, decreaseLearningRate, l2RegularizerWeight, numIterations);
+        }
+
+        /// <summary>
+        /// Predict a target using a linear binary classification model trained with the AveragedPerceptron trainer, and a custom loss.
+        /// </summary>
+        /// <param name="ctx">The binary classification context trainer object.</param>
+        /// <param name="options">Advanced arguments to the algorithm.</param>
```

Comment:
>arguments [](start = 43, length = 9)

options, here and in the other ones.

Reply:
You have it in your list of things to do :)
Up to you, as long as it gets done. 

---
In reply to: [248787187](https://github.com/dotnet/machinelearning/pull/2163#discussion_r248787187) [](ancestors = 248787187,248543101)

Reply:
I added it to description of #2154

---
In reply to: [248835228](https://github.com/dotnet/machinelearning/pull/2163#discussion_r248835228) [](ancestors = 248835228,248787187,248543101)